service/release.go

```diff
@@ -36,10 +51,33 @@ func (s *ReleaseService) Create(ctx context.Context, input model.CreateReleaseIn
 	}
 
 	// TODO check if release name is unique per project
-	// TODO send slack notification
 	if err := s.repo.Create(ctx, rls); err != nil {
 		return model.Release{}, err
 	}
 
+	if sendReleaseNotification {
+		s.sendReleaseNotification(ctx, p, rls)
+	}
+
 	return rls, nil
 }
+
+func (s *ReleaseService) sendReleaseNotification(ctx context.Context, p model.Project, rls model.Release) {
+	if !p.IsSlackChannelSet() {
+		slog.Debug("notification not sent: slack channel missing for project", "project_id", p.ID)
+		return
+	}
+
+	tkn, err := s.settingsGetter.GetSlackToken(ctx)
+	if err != nil {
+		// when fetching slack token fails, just log the error
+		// fail attempt to send Slack notification should not affect the release creation
+		// two possible reasons for failure:
+		// 1. slack integration is not set (logged in debug level, as it's not an error, but possible scenario)
+		// 2. failed to fetch slack token (logged in error level)
+		slog.Log(ctx, apierrors.GetLogLevel(err), "failed to get slack token", "err", err)
```

Comment:
It is outside of the MVP scope, but it would be helpful to send an informational email to the user. This email would notify them that the app received a request to send a Slack notification, but it could not be delivered due to x.